worlds/sc2/locations.py

```diff
@@ -1099,6 +1105,8 @@ def get_locations(world: Optional['SC2World']) -> Tuple[LocationData, ...]:
                      lambda state: logic.the_reckoning_requirement(state)),
         make_location_data(SC2Mission.THE_RECKONING.mission_name, "Odin", SC2HOTS_LOC_ID_OFFSET + 2004, LocationType.EXTRA,
                      lambda state: logic.the_reckoning_requirement(state)),
+        make_location_data(SC2Mission.THE_RECKONING.mission_name, "Trash the Odin Early", SC2HOTS_LOC_ID_OFFSET + 2005, LocationType.SPEEDRUN,

```

Comment:
Would be nice if these location names still gave a time limit in minutes / seconds.

Reply:
Unfortunately the Odin gets sent at different times depending on difficulty.
Plus, it's not on a global timer, so there is a bit of variability depending on if you manage to skip certain dialogues at the start of the mission/how soon you spawn in Jim Raynor.

Reply:
Can unupgraded Roach + Hydra do it with banana economy?

Reply:
Honestly not sure. I believe on Brutal difficulty you get around 22.5 minutes to complete this objective. Given that `logic.the_reckoning_requirement` is equivalent to `zerg_competent_comp and zerg_competent_anti_air` I think this should be fine, but if others have different suggestions feel free to post what I should replace it with.